meshmode/array_context.py

```diff
@@ -255,8 +256,18 @@ class PytestPyOpenCLArrayContextFactoryWithHostScalars(
     force_device_scalars = False
 
 
+class PytestPytatoPyOpenCLArrayContextFactory(
+        _PytestPytatoPyOpenCLArrayContextFactory):
+
+    @property
+    def actx_class(self):
+        return PytatoPyOpenCLArrayContext
+
+
 register_pytest_array_context_factory("meshmode.pyopencl",
         PytestPyOpenCLArrayContextFactory)
+register_pytest_array_context_factory("meshmode.pytato_cl",
+        PytestPytatoPyOpenCLArrayContextFactory)
 register_pytest_array_context_factory("meshmode.pyopencl-deprecated",
         PytestPyOpenCLArrayContextFactoryWithHostScalars)
```

Comment:
```suggestion
register_pytest_array_context_factory("meshmode.pyopencl-deprecated",
        PytestPyOpenCLArrayContextFactoryWithHostScalars)
register_pytest_array_context_factory("meshmode.pytato_cl",
        PytestPytatoPyOpenCLArrayContextFactory)
```
(to keep all the pyopencls together)